.travis.yml

```diff
@@ -1,16 +1,16 @@
 language: csharp
 mono: latest
-dotnet: 2.1
+dotnet: 5.0
 sudo: required
 env:
   global:
     - NUODB_HOME=/opt/nuodb
     - NUO_ADD_DOMAIN_PASS=yes
     - NUO_START_AGENT=yes
   matrix:
-    - NUO_VERSION=3.2.5.1
-    - NUO_VERSION=3.3.2.2
-    - NUO_VERSION=3.4.1.1
+    - NUO_VERSION=4.0.8.2
```

Comment:
I think we should just remove this.  We don't support 4.0 anymore.

It's not a requirement that we have 3 versions to test :)